contrib/packs/actions/install.meta.yaml

```diff
@@ -17,6 +18,10 @@
       type: "string"
       default: "actions,aliases,sensors,triggers"
       description: "Possible options are all, triggers, sensors, actions, rules, aliases."
+    upgrade:
+      type: "boolean"
+      default: false
+      description: "Upgrade the pack if already installed."
```

Comment:
Fair. They do stay in that state though 😛 

Let's make sure we get to that issue after the release, it's pretty important.